vortexasdk/endpoints/vessel_summary.py

```diff
@@ -0,0 +1,151 @@
+
+        # Returns
+        List of vessel summaries matching the search arguments.
+
+        # Examples
+
+        - Let's find all summaries for all LNG and VLCC_PLUS vessels, from the week prior to October 31st, 2023.
+
+        ```python
+        >>> from vortexasdk import VesselSummary
+        >>> vessel_summary_df = VesselSummary().search(vessel_class=['lng', 'vlcc_plus'], timestamp='2023-10-31T23:59:59.000Z').to_df(columns=['vessel_id', 'timestamp', 'lat', 'lon', 'speed', 'heading', 'declared_destination', 'draught'])
+
+        ```
+        |    | vessel_id        |     lat  |      lon   | timestamp                | speed  | heading | declared_destination | draught |
+        |---:|:----------------:|---------:|-----------:|:-------------------------|-------:|--------:|---------------------:|--------:|
+        |  0 | bc49bed3d600b394 | 17.36560 | -161.39080 | 2023-10-31T23:57:11.000Z | 19.299 | 308     | >JP KZU XX           | 11      |
+        |  ...to >800 results
+
+        Note that we will show you all fields by default if you don't set the columns argument.
+        ```
+
+        - Now let's find summaries for vessels carrying crude, using the Vessels and Product Reference endpoints
+
+        ```python
+        >>> from vortexasdk import Vessels, Products
+        >>> crude = [p.id for p in Products().search(term="crude").to_list() if 'group' in p.layer]
+        >>> vessels_list = Vessels().search(vessel_product_types=crude).to_list()
+        >>> vessel_ids = [v.id for v in vessels_list]
+        >>> crude_summaries = VesselSummary().search(vessel_id=vessel_ids).to_df()
+
+
+        ```
+
+        # Further Documentation
+
+        [VortexaAPI Vessel Summary Reference](https://docs.vortexa.com/reference/POST/signals/vessel-summary)
+
+        """
+        api_params: Dict[str, Any] = {
+            "vessel_id": convert_to_list(vessel_id),
+            "vessel_class": [v.lower() for v in convert_to_list(vessel_class)],
+            "timestamp": timestamp,
+        }
+
+        response = super().search_with_client(**api_params)
+
+        return VesselSummaryResult(records=response["data"])
+
+
+AVAILABLE_VESSEL_CLASSES = [
```

Comment:
Should we keep this list? List of available VCs is already provided above

Reply:
Yeah lets remove this and keep the one in vessels